app/components/mandatarissen/mandataris-card.hbs

```diff
@@ -1,116 +1,130 @@
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Status</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
+      >{{this.status}}</div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Publicatie status</div>
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        <AuPill
+          @skin="link"
+          @draft={{false}}
+          @size="regular"
+          @icon="info-circle"
+          @iconAlignment="left"
+          @hideText={{false}}
+        >
+          {{if
+            @mandataris.publicationStatus
+            @mandataris.publicationStatus.label
+            "Bekrachtigd"
+          }}
+        </AuPill>
+        {{! <AuLinkExternal href="" class="au-u-margin-left-tiny">Bekijk bewijs</AuLinkExternal> }}
+        {{! <p
+          class="au-u-italic au-u-h-functional au-u-flex au-u-flex--vertical-center"
+        >
+          <AuIcon @icon="info-circle" />
+          Om een bekrachtiging terug te tekken moet je de
+          <AuLinkExternal href="">
+            technische dienst contacteren
+          </AuLinkExternal>
+        </p> }}
       </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Startdatum</div>
-        <div class="au-c-description-value">
-          {{moment-format @mandataris.start "DD-MM-YYYY"}}
-        </div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Rangorde</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
+      >{{@mandataris.rangorde}}</div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Start periode</div>
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        {{moment-format @mandataris.start "DD-MM-YYYY"}}
       </div>
-      <div class="au-o-grid__item au-u-1-2 au-u-1-3@medium">
-        <div class="au-c-description-label">Einddatum</div>
-        <div class="au-c-description-value">
-          {{#if @mandataris.einde}}
-            {{moment-format @mandataris.einde "DD-MM-YYYY"}}
-          {{else}}
-            heden
-          {{/if}}
-        </div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Fractie</div>
+      <div
+        class="au-c-description-value au-u-margin-top-tiny"
+      >{{this.fractie}}</div>
+    </div>
+    <div class="au-o-grid__item au-u-1-2">
+      <div class="au-c-description-label">Einde periode</div>
+      <div class="au-c-description-value au-u-margin-top-tiny">
+        {{#if @mandataris.einde}}
+          {{moment-format @mandataris.einde "DD-MM-YYYY"}}
+        {{else}}
+          heden
+        {{/if}}
       </div>
+    </div>
+    <div class="au-o-grid__item au-u-5-6">
+      <div class="au-c-description-label"><div>Beleidsdomein(en)</div></div>
```

Comment:
can we only show this if the mandataris classificatie is schepen or burgemeester or aangestelde burgemeester or districtsburgemeester?

Reply:
nvm about this one, should be tackled by another pr